tnm-mw-core-connector/src/api-spec/core-connector-api-spec-dfsp.yml

```diff
@@ -162,7 +162,7 @@ paths:
           description: Bad request
 
   /callback:
-    put:
+    post:
```

Comment:
hi @elijah0kello , why change to post? 

Typically Mojaloop's callbacks all use PUT method, is there a justification to make this change?

Reply:
That callback is for the DFSP to send us a transfer notification. The DFSP sends it in POST and not PUT like we earlier had built